src/Modal/Modal.js

```diff
@@ -73,12 +75,14 @@ const ModalContent = createComponent({
   `,
 });
 
-function Modal({ children, title, animationDuration, showClose, onClose, open, ...props }) {
+function Modal({ children, title, animationDuration, showClose, onClose, open, blur, blurTarget, ...props }) {
   const [isOpen, setOpen] = useState(open);
+  const blurNode = useRef();
```

Comment:
We could initialize the ref value here:
```suggestion
  const blurNode = useRef(document.querySelector(blurTarget));
```